demo/index.html

```diff
@@ -35,18 +35,34 @@
 <body>
     <input type="button" value="Share it" class="the-share-button" onclick="shareTheLink()" />
 
-    <script src="https://unpkg.com/share-api-polyfill/dist/share-min.js"></script>
+    <script src="../dist/share-min.js"></script>
```

Comment:
I do believe this will break the demo. Just need to load the script from the CDN.

```suggestion
    <script src="https://unpkg.com/share-api-polyfill/dist/share-min.js"></script>
```